### DISCUSSION TITLE
How to set the Bar Gauge color by specifying conditions

### DISCUSSION THREAD

**Discussion by Wannapayeot** [frangoteam/FUXA#1276] [Q&A]
I want to set the Bar Gauge color by assigning a condition to the value.
For example:
If value < 20, set color to red.
If value >= 20 and value < 50, set color to yellow.
If value >= 50, set color to green.
Can FUXA be set up? Can you recommend? Please.

**Comment by Wannapayeot** [comment]
![Screenshot 2024-07-16 164922](https://github.com/user-attachments/assets/9a6cfaa8-f264-4ace-ae3f-d5fe319de27f)

**Comment by robsori** [comment]
Hi. Same discussion here: https://github.com/frangoteam/FUXA/issues/609
